python/dgl/data/citation_graph.py

```diff
@@ -51,6 +51,8 @@ class CitationGraphDataset(DGLBuiltinDataset):
         A transform that takes in a :class:`~dgl.DGLGraph` object and returns
         a transformed version. The :class:`~dgl.DGLGraph` object will be
         transformed before every access.
+    reorder : bool
+        Whether to reorder the graph. Default: False.
```

Comment:
"Whether to reorder the graph" -> "Whether to reorder the graph using :func:`~dgl.reorder_graph`"